zcash_client_backend/src/data_api/wallet.rs

```diff
@@ -311,6 +341,9 @@ where
 /// [`sapling::OutputProver`]: sapling::prover::OutputProver
 #[allow(clippy::too_many_arguments)]
 #[allow(clippy::type_complexity)]
+#[deprecated(
```

Comment:
@str4d should we go ahead and deprecate `spend` at this point?

Reply:
Yeah, let's do that in this release. Add this to the changelog.